src/Brick/Widgets/ProgressBar.hs

```diff
@@ -38,16 +39,41 @@ progressBar :: Maybe String
             -- ^ The progress value. Should be between 0 and 1 inclusive.
             -> Widget n
 progressBar mLabel progress =
+    progressBar' mLabel progress (' ', ' ')
+
+-- | Draw a progress bar with the specified (optional) label,
+-- progress value and a pair of characters to fill the progress. 
+-- This fills available horizontal space and is one row high.
+progressBar' :: Maybe String
+             -- ^ The label. If specified, this is shown in the center of
+             -- the progress bar.
+             -> Float
+             -- ^ The progress value. Should be between 0 and 1 inclusive.
+             -> (Char, Char)
+             -- ^ Pair of characters to fill the bar. First character is used 
+             -- to fill the completed part, second character draws the uncompleted part.
+             -- Please be aware of using wide characters in Brick, 
+             -- see [Wide Character Support and the TextWidth class](https://github.com/jtdaugherty/brick/blob/master/docs/guide.rst#wide-character-support-and-the-textwidth-class)
+             -> Widget n
+progressBar' mLabel progress (completeChar, uncompleteChar) =
```

Comment:
Please rename `uncompleteChar` to `incompleteChar`, and similarly with `un` -> `in` in the variable names below.